JPS_Ontology/ontology/ontoheatnetwork/ontoheatnetwork.csv

```diff
@@ -1,90 +1,64 @@
-http://www.ontology-of-units-of-measure.org/resource/om-2/symbol,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,string,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,symbol
-http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl#hasCost,Object Property,,,HeatGenerator UNION MunicipalUtility,http://www.ontology-of-units-of-measure.org/resource/om-2/Cost,,A relationship between heat generator and various cost associated with it,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl,has cost
-http://www.theworldavatar.com/ontology/ontocape/upper_level/system.owl#isOwnerOf,Object Property,,,MunicipalUtility,HeatGenerator UNION http://www.theworldavatar.com/ontology/ontopowsys/PowSysRealization.owl#ThermalEnergyStorage,,A relationship between a company and the entities it owns,http://www.theworldavatar.com/ontology/ontocape/upper_level/system.owl,is owner of
-http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl#hasRevenue,Object Property,,,GasTurbine,CoGenRevenueInTimeInterval,,A relationship between gas turbine and its co-generated electricity revenue ,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl,has revenue
-http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl#hasIndividualCO2Emission,Object Property,,,HeatGenerator ,CO2EmissionInTimeInterval,,A relationship between heat generator/plant item and CO2 emission per time interval (in specific time interval),http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl,has individual CO2 emission
-http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl#hasFuelType,Object Property,,,Heat Generator,http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl#FuelType,,A relationship between heat generator and its fuel type,http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl,has fuel type
+http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#ThermodynamicStateProperty,Class,http://www.theworldavatar.com/ontology/ontocape/upper_level/system.owl#Property,IS-A,,,,A phase system property that can serve as an intensive thermodynamic state variable,https://www.theworldavatar.com/kg/ontoheatnetwork/,Thermodynamic state property
+http://www.ontology-of-units-of-measure.org/resource/om-2/Cost,Class,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl#Costs,EQUIVALENT-TO,,,,Includes all costs associated with heat generation,https://www.theworldavatar.com/kg/ontoheatnetwork/,Cost
```

Comment:
Since it's only those 5 definitions, I believe there is not much added value of including them in OM. Hence, I believe these use case/domain specific additions can remain here (as approved previously)